docs/Guides/v1_v2_migration.md

```diff
@@ -20,3 +20,22 @@ production, you should:
 1. Duplicate the `basistheory_reactor` with the same `code` as the corresponding `basistheory_reactor_formula`
 2. Change your production systems to start using the newly created `basistheory_reactor`
 3. Delete the old reactor
+
+## basistheory_application `create_key` and basistheory_application_keys
```

Comment:
Where do these show up?

Reply:
not sure what you mean?

Reply:
In the repo only. But you are fixing that now. Thanks.